Modules/MUON/MID/include/MID/ClustQcCheck.h

```diff
@@ -31,10 +31,18 @@ class ClustQcCheck : public o2::quality_control::checker::CheckInterface
   ~ClustQcCheck() override = default;
 
   // Override interface
+  void configure() override;
   Quality check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap) override;
   void beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult = Quality::Null) override;
   std::string getAcceptedType() override;
 
+ private:
+  float mClusterTF = 0;
+  int mOrbTF = 32;
+  // float scaleTime = 0.0114048; // 128 orb/TF * 3564 BC/orb * 25ns
+  float scaleTime = 0.0000891; // 3564 BC/orb * 25ns
+  float mClusterScale = 100;
+
   ClassDefOverride(ClustQcCheck, 2);
```

Comment:
I am not sure, but should the class version be increased when the private members are added ?

Reply:
I dived into ROOT's documentation and it says that it has to be increased: 
>  Every time you change the data members of a class, you should increase its ClassVersionID by one.